app/src/main/res/layout/activity_recipe.xml

```diff
@@ -187,13 +187,209 @@
             app:layout_constraintStart_toEndOf="@+id/nbServings"
             app:layout_constraintTop_toBottomOf="@+id/ingredientsIcon" />
 
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/nutritionalValueCard"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginBottom="16dp"
+            android:layout_marginStart="1dp"
+            app:cardCornerRadius="5dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@+id/recipePicture"
+            app:layout_constraintTop_toBottomOf="@+id/ingredientsList"
+            app:strokeColor="@color/stroke_color"
+            app:strokeWidth="1dp">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginVertical="16dp">
+
+                <ImageView
+                    android:id="@+id/nutritional_info_icon"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="10dp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:srcCompat="@android:drawable/ic_menu_info_details"
+                    app:tint="@color/purple_700" />
+
+                <TextView
+                    android:id="@+id/nutritional_values_title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="10dp"
+                    android:text="@string/recipe_nutritional_value_title"
```

Comment:
for the textColor, it is better to create a color in the resource file instead of hardcoding it